web-proxy/molecule.yml

```diff
@@ -0,0 +1,26 @@
+# Syntax check only
+---
+dependency:
+  name: galaxy
+  requirements_file: ../requirements.yml
+
+driver:
+  name: docker
+
+docker:
+  containers:
+  - name: host-populate
```

Comment:
I don't think this matters since it's only a syntax check, but this should ideally match the hosts in the playbook: `prod-web-proxies`